src/styles/watch.scss

```diff
@@ -0,0 +1,66 @@
+.stopwatch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  margin: 0 auto;
+  width: 80vmin;
+  height: 80vmin;
+  background-color: aquamarine;
+
+  &__face {
+    display: flex;
+    justify-content: center;
+    border: 3px dotted black;
+    width: 400px;
+    height: 400px;
+    border-radius: 50%;
+  }
+
+  &__big-hand {
+    position: absolute;
+    z-index: 1;
+    height: 200px;
+    width: 5px;
+    background-color: blue;
+  }
+
+  &__small-hand {
+    position: absolute;
+    height: 100px;
+    width: 9px;
+    margin: 100px;
+    background-color: red;
+  }
+
+  &__center {
+    position: absolute;
+    align-self: center;
+    z-index: 2;
+    width: 20px;
+    height: 20px;
+    background-color: chocolate;
+    border-radius: 50%;
+  }
+
+  &:hover {
+    .stopwatch__big-hand {
+      animation: rotate 60s linear infinite;
+      transform-origin: bottom;
+    }
+
+    .stopwatch__small-hand {
+      animation: rotate 720s linear infinite;
```

Comment:
<img width="525" alt="Screenshot 2023-08-21 at 18 52 58" src="https://github.com/mate-academy/layout_stop-watch/assets/135232001/a4cd02fc-3818-4635-9b82-b7401b5d00e5">
This doesn't match the task requirements